mantidproject/mantid
# Fix for performance regression in ScanningWorkspaceBuilder

Fix regression affecting ScanningWorkspaceBuilder performance.

**To test:**

Check all tests pass. If you want to check the regression is fixed try with and without this fix. There should be a speedup, for me it was 26 s -> 14 s.

Fixes #20468. 

*Does not need to be in the release notes.* - these changes were introduced recently.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
